hagglehaul.client/src/Signin.css

```diff
@@ -0,0 +1,91 @@
+
+.signup-container {
+    max-width: 400px;
+    margin: auto;
+    padding: 20px;
+    border: 1px solid #ddd;
+    border-radius: 5px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    background-color: #fff;
+  }
+  
+  h2 {
+    font-size: 24px;
+    margin-bottom: 20px;
+  }
+  
+  form {
+    display: flex;
+    flex-direction: column;
+  }
+  
+  label {
+    margin-top: 10px;
+  }
+  
+  input {
+    padding: 10px;
+    margin-bottom: 10px;
+    border: 1px solid #ddd;
+    border-radius: 3px;
+    transition: border-color 0.3s;
+  }
+  
+  input.valid {
+    border-color: green;
+  }
+  
+  input.invalid {
+    border-color: red;
+  }
+  
+  .check-mark,
+  .cross-mark {
+    display: inline-block;
+    margin-left: 5px;
+    font-size: 16px;
+  }
+  
+  .check-mark::before {
+    content: '\2713'; 
+    color: green;
+  }
+  
+  .cross-mark::before {
+    content: '\2717'; 
+    color: red;
+  }
+  
+  p {
+    font-size: 14px;
+    color: red;
+    margin-top: 5px;
+  }
+  
+  select {
+    padding: 10px;
+    margin-bottom: 10px;
+  }
+  
+  button {
```

Comment:
Please do not style global elements unless you intend to style them globally. This will apply the style to all buttons. Please use classes, e.g., `.login-button` and `<button className="login-button" />` to style components. 

Please change this for all elements you do not intend to style globally (p, select, input, etc).